### PR TITLE
Lizard Firebreath is now a cone of fire instead of a fireball

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

https://user-images.githubusercontent.com/51863163/135035875-5246ca7c-388d-4cc9-9773-6a2076adf463.mp4

This PR reworks the firebreath mutation from being a wizard fireball that one hit crits with a power chromosome to a wave of fire.

The cone is 3 tiles, or 5 with the power chromosome, and travels in mostly a three wide column.

The closer you are to the caster, the more fire damage you take. 

- [x] 2 wide cones are super wide for some reason, need to check that out.

## Why It's Good For The Game

Non-antags can get wizard fireball and that's really dumb, this re-flavors it to be more like a dragon's breath, while removing the stupid aspect.

## Changelog

:cl: Melbert
balance: The firebreath Mutation is now a cone of fire, instead of a real fireball. 
/:cl:

### CODE REVIEW

File: code/datums/mutations/actions.dm [hunk 1]
```diff
@@ -104,59 +104,59 @@
...
+	to_chat(our_lizard, span_warning("Something in front of your mouth caught fire!"))
+
+/obj/effect/proc_holder/spell/cone/staggered/firebreath/do_turf_cone_effect(turf/target_turf, level)
+	// Further turfs experience less exposed_temperature and exposed_volume
+	new /obj/effect/hotspot(target_turf)
+	target_turf.hotspot_expose(max(500, 900 - (100 * level)), max(50, 200 - (50 * level)), 1)
+
+/obj/effect/proc_holder/spell/cone/staggered/firebreath/do_mob_cone_effect(mob/living/target_mob, level)
+	// Further out targets take less burn damage
+	target_mob.apply_damage(max(10, 30 - (5 * level)), BURN)
```
Comment (ATH1909) [inline code/datums/mutations/actions.dm:153]: ngl, this is a really disappointing amount of damage

Comment (MrMelbert) [reply code/datums/mutations/actions.dm:153]: In close range, it's on par with the space dragon's fire breath, which is what the damage was based off of. It also applies firestacks + the hotspot, so it causes a good about of residual burns.

Comment (EricZilla) [reply code/datums/mutations/actions.dm:153]: Also consider its gone from single target to "Nukies all dead"

**Comment by necromanceranne** [comment]
I cast burning hands.

**Comment by InvalidArgument3** [comment]
Do you get set on fire if you're moving in the direction of your fire breath attack?

File: code/datums/mutations/actions.dm (hunk 1 above)
Comment (ATH1909) [reply code/datums/mutations/actions.dm]: > In close range, it's on par with the space dragon's fire breath, which is what the damage was based off of. It also applies firestacks + the turf hotspot, so it causes a good about of residual burns.

yeah, but space dragons aren't at risk of catching on fire from the people they light on fire with their own fire breath

Comment (AMonkeyThatCodes) [reply code/datums/mutations/actions.dm]: ignore ath he is a powergamer

**Comment by ExcessiveUseOfCobblestone** [comment]
In the future any changes to species specific interactions should be approved prior

**Comment by rellyreri** [comment]
does this fix non lizards being able to have it

**Comment by MrMelbert** [comment]
> 
> 
> In the future any changes to species specific interactions should be approved prior

image

Forgot to include it since I wasn't sure if it counted or not but mothblocks said it's fine

**Comment by ExcessiveUseOfCobblestone** [comment]
foxpoggers

**Comment by TemporalOroboros** [comment]
Question. Can you make it give the user a little nudge backwards? One of the niche uses of fire-breathing was 0-g propulsion in a pinch.

**Comment by MrMelbert** [comment]
> 
> 
> Question. Can you make it give the user a little nudge backwards? One of the niche uses of fire-breathing was 0-g propulsion in a pinch.

yeah sure

**Pull request merged**